ONSdigital/rm-case-service
# Adding metadata to caseevents

# Motivation and Context
In response-operation-ui, we want to be able to see the respondent name on the responses status page when a external user has completed a collection exercise. For this to happen, a new column needs to be added to the caseevents table which will hold metadata required to get the respondent name.

# What has changed
- Added a new column to case events table

# How to test?
- Run `mvn clean install` and it should build successfully
- Run with the [case-svc-api PR](https://github.com/ONSdigital/rm-casesvc-api/pull/33), [R-ops branch](https://github.com/ONSdigital/response-operations-ui/tree/us211-display-respondent-for-seft-rus-ce-in-response-status), [Frontstage branch](https://github.com/ONSdigital/ras-frontstage/tree/us211-display-respondent-for-seft-rus-ce-in-response-status) and complete a SEFT collection exercise. When completed, look on response ops response status page and the completed collex should have a respondent name attached to it.
- Run with acceptance tests

# Links
[Trello](https://trello.com/c/1rcfmtok)
